blockly-ws-search/webpack.config.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/**
+ * @fileoverview Webpack configuration file.
+ * @author aschmiedt@google.com (Abby Schmiedt)
+ */
+
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = env => {
+    return {
+        target: 'web',
+        mode: env.mode,
+        entry: './src/WorkspaceSearch.js',
+        devtool: 'source-map',
+        output: {
+            path: path.resolve(__dirname, 'dist'),
+            filename: 'workspace-search.umd.js',
+            libraryTarget: 'umd',
+            globalObject: 'this'
```

Comment:
Since you're referencing the build file ``../dist/workspace-search.umd.js`` from your test ``index.html`` you'll need to tell webpack where to serve its build files when they're in-memory (ie: when webpack-dev-server) is running. You can do that by adding: 
```
publicPath: '/dist/'
```

Reply:
To repro the issue, try running ``npm run clean`` and then ``npm start``.

Reply:
sweet thanks!